pyproject.toml

```diff
@@ -61,35 +60,6 @@ graphviz = { version = "^0.20.1" }
 dowhy = { version = "^0.8" }
 ipython = { version = "^7.4.0" }
 nbsphinx = { version = "^0.8" }
-# [tool.poetry.extras]
```

Comment:
I see, so the sphinx-related dev-dependencies are always going to install in CI. That's fine - in dowhy we did that slightly differently but this should hold until poetry 1.2.